src/pal/src/include/pal/process.h

```diff
@@ -43,6 +44,11 @@ extern DWORD gSID;
 
 extern LPWSTR pAppDir;
 
+// The Mac sandbox application group ID (if exists) and container (shared) path
+extern LPCSTR gApplicationGroupId;
```

Comment:
A nit - the convention we use in PAL is to put underscore after the `g` - so it would be `g_ApplicationGroupId` for this one

Reply:
Yeah,  I was trying to be consistent with the above. gAppDir,  gPID,  etc.

Reply:
Ah, hmm, I can see we are more inconsistent in various parts of the PAL than I've thought. Then please keep it the way you did it, consistency with the surrounding code is more important than consistency with stuff in different places of the PAL.